vscode/src/autoedits/autoedits-provider.ts

```diff
@@ -97,10 +98,12 @@ export class AutoeditsProvider implements vscode.InlineCompletionItemProvider, v
         contextRankingStrategy: ContextRankingStrategy.TimeBased,
         dataCollectionEnabled: false,
     })
+    private readonly statusBar: CodyStatusBar
 
     constructor(
         chatClient: ChatClient,
         fixupController: FixupController,
+        statusBar: CodyStatusBar,
```

Comment:
⛳ 
```suggestion
        private readonly statusBar: CodyStatusBar,
```
Will automatically declare the field above and set it in the constructor. But what you have is very readable.

Reply:
Good to know! Will leave as is to match the style of the other fields.